src/main/java/org/jenkinsci/plugins/github_branch_source/BranchSCMHead.java

```diff
@@ -47,4 +52,23 @@ public BranchSCMHead(@NonNull String name) {
     public String getPronoun() {
         return Messages.BranchSCMHead_Pronoun();
     }
+
+    @Restricted(NoExternalUse.class)
+    @Extension
+    public static class MigrationImpl extends SCMHeadMigration<GitHubSCMSource, SCMHead, AbstractGitSCMSource.SCMRevisionImpl> {
```

Comment:
Do not get me started on all the mess I have had to clean up. Without this there is a storm of builds and people don't want a storm of builds even if it is good for them in the long term... and that is why we cannot have nice things